imports/plugins/included/sms/server/methods/sms.js

```diff
@@ -68,7 +68,8 @@ Meteor.methods({
     // check for addressBook phone
     if (user && addressBook) {
       if (addressBook[0].phone) {
-        phone = addressBook[0].phone;
+        const [phoneData] = addressBook;
```

Comment:
This borders on less readable.. I think I'll leave this one for now, but not sure I like where this is going.

Reply:
Proposing an alternative here ... #3644